### ISSUE TITLE
Bottts iOS Crash

### ISSUE THREAD

**Issue by NilsBaumgartner1994** [dicebear/dicebear#230]
Some bottts avatars work fine, others don’t.
For example an avatar with the seed: „Test“ works, while with „Tests“ it crashes.

i will forward if it is the svg rendering library.

**Comment by NilsBaumgartner1994** [comment]
```
Error: Text strings must be rendered within a <Text> component.

This error is located at:
    in RNSVGGroup (at G.tsx:33)
    in G (at xml.tsx:229)
    in RNSVGGroup (at G.tsx:33)
    in G (at xml.tsx:229)
    in RNSVGGroup (at G.tsx:33)
    in G (at xml.tsx:229)
    in RNSVGGroup (at G.tsx:33)
    in G (at Svg.tsx:202)
    in RNSVGSvgView (at Svg.tsx:197)
    in Svg (at xml.tsx:103)
    in SvgAst (at xml.tsx:118)
    in SvgXml (at Avatar.tsx:120)
    in RCTView (at View.js:34)
    in View (at ParentDimension.tsx:12)
    in ParentDimension (at Avatar.tsx:126)
    in RCTView (at View.js:34)
    in View (at Rectangle.tsx:15)
    in RCTView (at View.js:34)
    in View (at Rectangle.tsx:13)
    in _default (at Avatar.tsx:125)
    in Avatar (at AvatarsExample.tsx:252)
...
```

**Comment by FlorianKoerner** [comment]
Hi Nils!

Please provide a "Minimal reproducible example". Preferably with https://snack.expo.dev/

I can't reproduce it with your description unfortunately.

**Comment by FlorianKoerner** [comment]
@NilsBaumgartner1994 @corbt

Can you please check if the bug has been fixed by chance with the current alpha version?

https://api.dicebear.com/5.x/bottts/svg?seed=foobar instead of https://avatars.dicebear.com/api/bottts/foobar.svg

**Comment by NilsBaumgartner1994** [comment]
I've got the problem by using react-native-svg

https://github.com/react-native-svg/react-native-svg/issues/1580

It seems that the svg is somehow broken itself. I get working some of the bots. I could provide you my whole project, so I cant provide a "Minimal" Project

**Comment by FlorianKoerner** [comment]
Thanks to the sample SVG you posted [here](https://github.com/react-native-svg/react-native-svg/issues/1580#issuecomment-1157990704), I was at least able to find the error in the SVG. I am now checking what the problem is and will publish a fix.

**Comment by FlorianKoerner** [comment]
Should be fixed with version 4.10.3.

https://github.com/dicebear/dicebear/commit/7deaa9b7cbe1f91c6a7f22223b009101ef9bdc53

**Comment by NilsBaumgartner1994** [comment]
Thanks 😊 


Beste Grüße,
Nils Baumgartner
